models/activityBlueprint.js

```diff
@@ -0,0 +1,18 @@
+import connector from "#models/databaseUtil"
+
+const activityBluePrintSchema = {
+    number : { type: Number, required: true },
+    academicYearYear: {
+        type: String,
+        required: true,
+        validate: {
+          validator: function(value) {
+            return /^2\d{3}$/.test(value); // Matches "2" followed by any 3 digits
+          },
+          message: props => `${props.value} is not a valid year format starting with "2"!`
+        }
+      },
+      type : { enum : [Odd, even] , required : true},
```

Comment:
```suggestion
      type : { enum : [Odd, even] , required : true},
```
add quotes for odd and even and make them uppercase